test/cql-pytest/run.py

```diff
@@ -260,6 +261,15 @@ def check_ssl_cql(ip):
     ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
     check_cql(ip, ssl_context)
 
+## Test that the Scylla REST API is serving, for wait_for_services() below.
```

Comment:
By the way, I was tempted to ask you to move this function to test/rest-api/run where you used it, but then I changed my mind - we should perhaps even use it in test/cql-pytest/run as well so make sure that the REST API is also up. Although maybe it's just a waste of time? I don't know.

Reply:
I think it's a good idea to use it in cql-pytest since it does use the api.
This will verify that the rest api is up and running along with the cql service.
It's possible (although highly unlikely) that a future change will start CQL before the REST API.

Reply:
Done in #9755